.github/workflows/conformance-ingress-default.yaml

```diff
@@ -0,0 +1,167 @@
+name: ConformanceIngress (Default Ingress)
+
+# Any change in triggers needs to be reflected in the concurrency group.
+on:
+  pull_request:
+    paths-ignore:
+      - 'Documentation/**'
+      - 'test/**'
+  push:
+    branches:
+      - master
+      - ft/master/**
+    paths-ignore:
+      - 'Documentation/**'
+      - 'test/**'
+
+permissions: read-all
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.event.after }}
+  cancel-in-progress: true
+
+env:
+  cilium_cli_version: v0.12.12
+  minikube_version: 1.28.0
+  timeout: 5m
+
+jobs:
+  ingress-conformance-test:
+    runs-on: ubuntu-latest
+    timeout-minutes: 120
+    steps:
+      - name: Checkout master branch to access local actions
+        uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8 # v3.1.0
+        with:
+          ref: ${{ github.event.repository.default_branch }}
+          persist-credentials: false
+      - name: Set Environment Variables
+        uses: ./.github/actions/set-env-variables
+
+      - name: Install Cilium CLI
+        run: |
+          curl -sSL --remote-name-all https://github.com/cilium/cilium-cli/releases/download/${{ env.cilium_cli_version }}/cilium-linux-amd64.tar.gz{,.sha256sum}
+          sha256sum --check cilium-linux-amd64.tar.gz.sha256sum
+          sudo tar xzvfC cilium-linux-amd64.tar.gz /usr/local/bin
+          rm cilium-linux-amd64.tar.gz{,.sha256sum}
+          cilium version
+
+      - name: Set image tag
+        id: vars
+        run: |
+          if [ ${{ github.event.pull_request }} ]; then
+            SHA=${{ github.event.pull_request.head.sha }}
+          else
+            SHA=${{ github.sha }}
+          fi
+          echo sha=${SHA} >> $GITHUB_OUTPUT
+
+      - name: Checkout
+        uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8 # v3.1.0
+        with:
+          ref: ${{ steps.vars.outputs.sha }}
+          persist-credentials: false
+
+      - name: Create minikube cluster with multiple nodes
+        uses: medyagh/setup-minikube@ab221dee176f8eabd8deddf849b5bf1d6244a6e8 # v0.0.11
+        with:
+          minikube-version: ${{ env.minikube_version }}
+          network-plugin: cni
+          cni: false
+          wait: false
+          start-args: --nodes 3 --docker-opt "default-ulimit=nofile=102400:102400"
```

Comment:
Is there any specific reason to use Minikube here instead of Kind? Not a problem at all but I'm wondering if the pipeline would not run faster on a Kind-based cluster.

Reply:
Happy to move it to kind (I was wondering the same thing as you...), but it builds upon the previous conformance tests which all use minikube, I only modified this one to use the default behavior

Reply:
Your call, if not worth the trouble it's OK as is, and consistency with the other conformance workflows is also good. Probably @sayboras just needed Minikube for the other workflows.

Reply:
I'll keep it minikube for now and will probably file an issue to consider kind (unless @sayboras has a reason)

Reply:
as discussed offline, we can tackle this separately, we might even use LB IPAM feature if possible.